test/cypress/cypress/integration/02-Blog/02-blog-tests.js

```diff
@@ -0,0 +1,32 @@
+/// <reference types="Cypress" />
+
+import faker from 'faker';
+import { blog } from '../../support/objects';
+
+describe('Blog Tests', function() {
+    it('Can add blog post and it displays in the frontend', function() {
+        cy.login(blog);
+        cy.visitAdmin(blog);
+        // blog admin menu
+        cy.get('li.icon-class-fa-rss > .item-label > .title').click();
+        // create blog post button
+        cy.get('p > .btn.btn-primary').click();
+
+        var blogName = faker.random.uuid();
+        // fill form
+        cy.get('#TitlePart_Title').type(blogName);
+        var text = faker.lorem.sentence();
+        cy.get('.CodeMirror textarea')
+            // we use `force: true` below because the textarea is hidden
+            // and by default Cypress won't interact with hidden elements
+            .type(text, { force: true });
+
+        // click publish button
+        cy.get('.btn-success').click();
+
+        cy.visitTenantPage(blog, `blog/${blogName}`);
+
+        cy.get('h1').should('contain.text', blogName);
+        cy.get(':nth-child(3) > .row > .col-lg-8 > p').should('contain.text', text);
```

Comment:
I really hate this selector. Would be even better to just check there is a <p> with the sentence inside, anywhere in the page. Also please use a static sentence, even if it has been generated randomly once. We have already talked about it. There is no advantage to generate a new one every time, it can only add flakyness, which is interesting to find new bugs, but not to ensure that the application isn't regressing like this one is aiming to.

I am not opposed to have tests that are not run continuously that would loop with a high number of random values to try to find ways to break the system. This is what tools like these do: https://github.com/Metalnem/sharpfuzz
And once an issue is found, a unit test can be filed for a specific value that triggers an issue.

Reply:
I hate this selector as well. I will add an id or something more specific at some point.